### DISCUSSION TITLE
Is there a desktop client on Mac for Mol* avaiable?

### DISCUSSION THREAD

**Discussion by TomHsiung** [molstar/molstar#1265] [General]
Hi, everyone

This tool is cool and I wonder if I can download this nice app and run it on my macOS as a desktop app. Another alternative is to delopy it on my web server so that I can access it via Internet. Thanks.

Tom

**Comment by dsehnal** [comment]
There is no official support, but it is possible to use Electron for this. Or you can just download the NPM package and run `index.html` from `build/viewer`.

**Reply by TomHsiung** [reply]
Thanks a bunch, pal. I will try the NPM package method in a few days. Later.

**Reply by TomHsiung** [reply]
I downloaded the Node.js installer for macOS on [https://nodejs.org/en/download/prebuilt-installer](Node.js). After the installation, I got the package of molstar by command,

> npm pack molstar

The package is small, about 7 Mbytes on my macOS disk. After unpacking, it takes about 34.2 Mbytes. Finally, I copied the whole folder to my web server directory and it works very well as the official molstar website viewer.

I really appreciate the work and the courteous contributions of all the developers for this great project!

Tom
